src/bin/flamegraph.rs

```diff
@@ -12,6 +12,10 @@ struct Opt {
     #[structopt(short = "p", long = "pid")]
     pid: Option<u32>,
 
+    /// Generate shell completions for the given shell.
```

Comment:
This should conflict with `--pid` or the use of trailing arguments. Can you change that?

Reply:
I tried it and it seems to just ignore `conflicts_with = "trailing_arguments"`. Right now `--completions` takes precedence over any other switch. If we add `conflicts_with = "pid"`, shouldn't we also add `conflicts_with` attributes for all the switches in `lib.rs/Options`, i.e. `--verbose`, `--output`, `--open`, `--root`, ...? Feel free to add that if you think it's necessary. I'll be on vacation soon and I don't think I'll manage before that.